src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/VideoBooking.kt

```diff
@@ -41,15 +47,60 @@ class VideoBooking private constructor(
 
   val createdBy: String,
 
-  val createdTime: LocalDateTime = LocalDateTime.now(),
+  val createdTime: LocalDateTime = now(),
 ) {
 
+  @OneToMany(mappedBy = "videoBooking", fetch = FetchType.LAZY, cascade = [CascadeType.ALL], orphanRemoval = true)
```

Comment:
Perhaps this doesn't need to be LAZY either, there will never be that many on a single booking.